src/probnum/diffeq/steprule.py

```diff
@@ -76,7 +76,7 @@ def __init__(
         firststep,
         atol,
         rtol,
-        limitchange=(0.1, 5.0),
+        limitchange=(0.2, 5.0),
```

Comment:
this asymmetry kind of bothered me (and the outputs seem not worse)... I suppose @NinaEffenberger will find that hilarious ;)

Reply:
...ok just so I understand, the only reason for this change is aesthetics? :joy: I think there is an argument for the previous setting though, like allowing the solver to slow down more quickly when encountering a steep slope or something, but then I also saw defaults of (0.2,10.0) in the Julia repos so not sure if that really counts.

Long story short, from my experience I wouldn't expect this to matter a lot so I'm 100% with this change :+1:

Reply:
What is the default in `ProbNumDiffeq.jl`?

Reply:
Decision: use (0.2, 10.0) (but without any strong reasoning, other than that it seems to be common values)